sass/_FacetVariables.scss

```diff
@@ -0,0 +1 @@
+$facet-value-padding: 20px;
```

Comment:
This variable was introduced in its own functional commit. It could easily be reverted if you don't like it.